src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -151,19 +153,25 @@ std::vector<uint64_t> LiveFunctionsDataView::GetFunctionTimerDurations(int row)
   return timer_durations;
 }
 
-void LiveFunctionsDataView::OnSelect(const std::vector<int>& rows) {
-  UpdateHighlightedFunctionId(rows);
-  UpdateSelectedFunctionId();
+void LiveFunctionsDataView::UpdateHistogram() { UpdateHistogram(GetVisibleSelectedIndices()); }
 
+void LiveFunctionsDataView::UpdateHistogram(const std::vector<int>& visible_selected_indices) {
```

Comment:
Why not having only one function (the one without parameter) and asking for the visible_selected_indices inside that function?

Reply:
@vickyliu-go4it makes the same point, see answer there.